cdk/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cdk",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "test": "jest",
+    "format": "prettier --write \"{lib,bin}/**/*.ts\"",
+    "lint": "eslint lib/** bin/** --ext .ts --no-error-on-unmatched-pattern",
+    "synth": "cdk synth --path-metadata false --version-reporting false",
+    "diff": "cdk diff --path-metadata false --version-reporting false"
+  },
+  "devDependencies": {
+    "@guardian/cdk": "43.0.0",
```

Comment:
Yep, when we are ready to merge I will use the latest `@guardian/cdk` and rebase the whole set of stacked PRs (I already rebased these PRs _a lot_ of times, so will wait until we're ready to ship to avoid repeating work)

Reply:
https://github.com/guardian/mobile-save-for-later/pull/64/commits/76a530ac3054453d03cc18dca6a44f1f5f4c4896